src/Package/Impl/DataInspect/VariableProvider.cs

```diff
@@ -54,19 +56,27 @@ internal class VariableProvider: IVariableDataProvider, IDisposable {
 
         public IRSession RSession { get; }
 
+        public bool Enabled { get; private set; }
```

Comment:
Are you going to access this property from different threads? And how is it different from `IRSession.IsHostRunning`?